src/kubectl.ts

```diff
@@ -147,16 +153,35 @@ async function kubectllInternalArgs(args: string[], namespace: string | undefine
 }
 
 export async function baseKubectlPath(): Promise<string> {
-	let result : shell.FindBinaryResult = await shell.findBinary('kubectl');
+	const result : FindBinaryResult = await findBinary('kubectl');
 	if (result && result.output && result.err === null) {
 		return result.output;
 	}
-	let bin = config.getActiveKubectlconfig();
+	const bin: string = config.getActiveKubectlconfig();
 	if (!bin) {
-		bin = 'kubectl';
-		return bin;
+		return 'kubectl';
 	}
-	let binpath = path.normalize(bin);
-	return binpath;
+	return path.normalize(bin);
 }
 
+export async function findBinary(binName: string): Promise<FindBinaryResult> {
+	let cmd: string = `which ${binName}`;
+
+	if (shell.isWindows()) {
+		cmd = `where.exe ${binName}.exe`;
+	}
+
+	const opts: any = {
+		async: true,
+		env: {
+			HOME: process.env.HOME,
+			PATH: process.env.PATH
+		}
+	};
+
+	const execResult: ShellResult = await shell.execCore(cmd, opts);
+	if (execResult.code === 0) {
+		return { err: null, output: execResult.stdout };
+	}
+	return { err: execResult.code, output: execResult.stderr };
+}
```

Comment:
I think you had an issue with conflict resolution during rebase. This part has been moved to the shell class

Reply:
crap. good catch.